### PR TITLE
COSA-984 / Rename Points to ProjectX

### PR DESCRIPTION
**Pull request by arielj**
### Jira Ticket 

https://ombulabs.atlassian.net/browse/COSA-984

### Motivation / Context

The app was still using `Points` in many places, now it shows ProjectX.

### QA / Testing Instructions

You can check the main screen and also run the tests to make sure everything is fine.

### Screenshots:

image
___

I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md).

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -58,4 +56,4 @@ If the issue has an effect in the frontend, include any relevant screenshots and
 
 Include relevant log snippets or files here.
 
-**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**
+**I will abide by the [code of conduct](https://github.com/fastruby/projectx/blob/main/CODE_OF_CONDUCT.md)**
```
Comment (JuanVqz) [inline .github/ISSUE_TEMPLATE/bug_report.md]: We don't have a code of conduct in this repo, probably, you can remove the line

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 2]
```diff
@@ -44,4 +37,4 @@ Include any mockup idea related to the requested feature if it applies.
 
 If you have resources related to the implementation or research for this feature, add them here.
 
-**I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md)**
+**I will abide by the [code of conduct](https://github.com/fastruby/projectx/blob/main/CODE_OF_CONDUCT.md)**
```
Comment (JuanVqz) [inline .github/ISSUE_TEMPLATE/feature_request.md]: Same

File: pull_request_template.md [hunk 3]
```diff
@@ -1,16 +1,20 @@
...
 
 ### Screenshots:
+
 <!-- Add screenshots (applicable to any UI changes) -->
 
-___
+---
 
-I will abide by the [code of conduct](https://github.com/fastruby/points/blob/main/CODE_OF_CONDUCT.md).
+I will abide by the [code of conduct](https://github.com/fastruby/projectx/blob/main/CODE_OF_CONDUCT.md).
```
Comment (JuanVqz) [inline pull_request_template.md]: same, please

**Comment by arielj** [comment]
@JuanVqz done

**Review by JuanVqz — APPROVED** [review]

**Pull request merged**